api/MediaStreamAudioSourceNode.json

```diff
@@ -110,10 +110,10 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/MediaStreamAudioSourceNode/mediaStream",
           "support": {
             "chrome": {
-              "version_added": "60"
+              "version_added": "14"
```

Comment:
What's the source of your version numbers?

Reply:
@jpmedley - I got the 14 from someone telling me it was version 14. I do not recall who it was. I agree though that it seems early. I think part of the problem is that this addition predates the move from SVN to git, so a lot of the methodologies I use to date things don't work correctly. Do you have recommendations for how I can sort out the Chrome version in cases like this?

Reply:
@jpmedley - I got the 14 from someone telling me it was version 14. I do not recall who it was. I agree though that it seems early. I think part of the problem is that this addition predates the move from SVN to git, so a lot of the methodologies I use to date things don't work correctly. Do you have recommendations for how I can sort out the Chrome version in cases like this?

Reply:
If I can't find its commit hash in [Omaha Proxy](https://omahaproxy.appspot.com/) or [Chromium Dash](https://chromiumdash.appspot.com/commits), I compare the earliest commit date I can find to the next **beta** data afterwords. I'll publish that data.

Reply:
@jpmedley - I'm going to just change this back to the 60 it was previously and assume that whoever added it (probably you) knew what they were doing. :)

Reply:
These version numbers came from the original added by Chris back when he first created the file. Hopefully @chrisdavidmills can answer where he got the numbers from. :)